contracts/estate/EstateRegistry.sol

```diff
@@ -0,0 +1,236 @@
+pragma solidity ^0.4.22;
+
+import "zeppelin-solidity/contracts/math/SafeMath.sol";
+import "zeppelin-solidity/contracts/token/ERC721/ERC721Token.sol";
+import "zeppelin-solidity/contracts/ownership/Ownable.sol";
+
+import "../metadata/MetadataHolderBase.sol";
+
+
+contract PingableDAR {
+  function ping() public;
+  function ownerOf(uint256 tokenId) public returns (address);
+  function safeTransferFrom(address, address, uint256) public;
+}
+
+
+// @nico TODO: Standalone Storage.
+// @nico TODO: Document functions.
+contract EstateRegistry is ERC721Token, Ownable, MetadataHolderBase {
+  using SafeMath for uint256;
+
+  PingableDAR public dar;
+
+  // From Estate to list of owned tokenIds (LANDs)
+  mapping(uint256 => uint256[]) public estateTokenIds;
+
+  // From tokenId (LAND) to its owner estateId of owned tokenIds
+  mapping(uint256 => uint256) public tokenIdEstate;
+
+  // From Estate to mapping of tokenId to index on the array above
+  mapping(uint256 => mapping(uint256 => uint256)) public estateTokenIndex;
+
+  // Metadata of the Estate
+  mapping(uint256 => string) internal estateData;
+
+  // Operators of the Estate
```

Comment:
We only have one operator, Should it be `Operator of the Estate`?

Reply:
Good catch!